ncm-iptables/src/main/perl/iptables.pm

```diff
@@ -830,80 +783,38 @@ sub find_rule {
     return $?;
 }
 
-##########################################################################
-
-##########################################################################
-
-##########################################################################
-sub Configure($$@) {
-##########################################################################
-
+sub Configure {
     my ($self, $config) = @_;
+    local $@;
```

Comment:
the old code was evil, i don not understand that ncm-ncd didn't give more errors when this was used without `local $@` (there is code above that sets `$@` on success, like, wtf.)